plugins/pixart-rf/fu-pxi-ble-device.c

```diff
@@ -91,33 +91,52 @@ fu_pxi_ble_device_prepare_firmware(FuDevice *device,
 				   FwupdInstallFlags flags,
 				   GError **error)
 {
-	FuPxiBleDevice *self = FU_PXI_BLE_DEVICE(device);
-	const gchar *model_name;
 	g_autoptr(FuFirmware) firmware = fu_pxi_firmware_new();
 
 	if (!fu_firmware_parse(firmware, fw, flags, error))
 		return NULL;
 
-	/* check is compatible with hardware */
-	model_name = fu_pxi_firmware_get_model_name(FU_PXI_FIRMWARE(firmware));
-	if ((flags & FWUPD_INSTALL_FLAG_FORCE) == 0) {
-		if (self->model_name == NULL || model_name == NULL) {
-			g_set_error_literal(error,
+	if (fu_device_has_private_flag(device, FU_PXI_DEVICE_FLAG_IS_HPAC) &&
+	    fu_pxi_firmware_is_hpac(FU_PXI_FIRMWARE(firmware))) {
+		g_autoptr(GBytes) fw_tmp = NULL;
+		gsize hpac_fw_size = 0;
+		const guint8 *fw_ptr = g_bytes_get_data(fw,NULL);
```

Comment:
you need to run clang-format -- you can use `find . -name '*.c' -o -name '*.h' -o -name '*.proto' | xargs clang-format -i` manually if that helps

Reply:
Already run clang-format in Revised in b623d0cd5cad59a43bc7c4016fe867bb8013caf6.